sklearn/utils/tests/test_extmath.py

```diff
@@ -568,6 +567,32 @@ def max_loading_is_positive(u, v):
     assert not v_based
 
 
+def test_randomized_svd_lapack_driver():
+    # Check that different SVD drivers provide consistent results
+
+    # Matrix being compressed
+    n = 123
+    m = 456
+    k = 10
+
+    X = np.random.rand(n, m)
```

Comment:
Let's seed the rng to make test outcome independent of test running order which makes rare failures hard to debug, especially when running tests in parallel with `pytest-xdist`.


```suggestion
    rng = np.random.RandomState(42)
    X = rng.rand(n, m)
```

Reply:
Done